cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -43,6 +43,13 @@ module Cardano.Api.IPC (
     QueryInEra(..),
     QueryInShelleyBasedEra(..),
     queryNodeLocalState,
+
+    -- *** Tip query
+    getLocalChainTip,
+
+    -- *** Helpers
+    ConsensusMode(..),
+    consensusModeOnly,
```

Comment:
In the end we should export these only via Cardano.Api.Mode, and that'll get picked up and re-exported by Cardano.Api. This is ok for now (though we can add a TODO to remind us if you like).